melanie-melkirschner/hw_2/3-dice-lab/js/app.js

```diff
@@ -0,0 +1,31 @@
+/*
+
+Creating a page where every time the page is loaded, the console logs random values for two dice. 
+
+1) Write down pseudocode for the program described below.
+
+2) Follow the steps in your pseudocode to write the code for your function.
+* generate a random number between 1 - 6 and store to a variable, firstDie
+* generate a random number between 1 - 6 and store to a variable, secondDie
+* log the values of the first and second dice to the console, labeling each value
+* log the total of the two dice to the console, with a label
+
+3) Create a statement that runs the function you created whenever the page loads.
+  (Hint: Reference when the page loads with the code window.onload, and set its value to the name of the function you created, without parentheses.)
+
+// */
+
+
+function rollDice()	 {	
+ var firstDie = Math.floor(Math.random() * 6) + 1;
+ var secondDie = Math.floor(Math.random() * 6) + 1;
+ 
+ var total = firstDie + secondDie;
+ console.log("The value of the first die is" + " " +firstDie);
+ console.log("The value of the second die is" +" " +secondDie);
+ console.log("The total value of the two dice is" +" " +total);
```

Comment:
Nice, but why not just add a trailing space after each string? Functionally it's the same and not a big deal in the grand scheme of things, it's just cleaner looking. :P